src/rlookup.c

```diff
@@ -581,7 +739,7 @@ static int getKeyCommonHash(const RLookupKey *kk, RLookupRow *dst, RLookupLoadOp
     // as it will hold the only reference to it after the next line.
     rsv = hvalToValue(val, kk->fieldtype);
     RedisModule_FreeString(RSDummyContext, val);
-  } else if (!strncmp(kk->name, UNDERSCORE_KEY, strlen(UNDERSCORE_KEY))) {
+  } else if (!strncmp(kk->path, UNDERSCORE_KEY, strlen(UNDERSCORE_KEY))) {
```

Comment:
PR descriptions says
> Implements the new API, without anyone using it.

Isn't this a change in an API that is currently being used?
If it is already being fixing now, should also be tested?

Reply:
I believe this is a bug. It can be moved to a different PR.
Seems like if one is trying to load the key name using `@__key`, it cannot be aliased to another name because of this.